soc/nordic/nrf54l/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ zephyr_include_directories(.)
 # We need a buffer in memory in a static location which can be used by
 # the KMU peripheral. The KMU has a static destination address, we chose
 # this address to be 0x20000000, which is the first address in the SRAM.
-if(NOT CONFIG_BUILD_WITH_TFM AND CONFIG_PSA_NEED_CRACEN_KMU_DRIVER)
+if(NOT CONFIG_BUILD_WITH_TFM AND CONFIG_PSA_NEED_CRACEN_KMU_DRIVER AND CONFIG_XIP)
```

Comment:
commit needs to be updated with `nrf-squash!` to fix original no-up commit

Reply:
Good point, I updated the commit